frontend/tests/components/jobs/result/utils/sections.test.jsx

```diff
@@ -306,7 +306,7 @@ describe("test JobActionsBar", () => {
       visualizersReportedList,
       visualizersToExecute,
     );
-    expect(result).toBe(2);
+    expect(result).toBe(1);
```

Comment:
Why 1 instead of 2 ? There are 2 visualizers (DNS and IP). Should it still be 2 ?

Reply:
Because the IP visualizer still has a page running